src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -45,6 +49,15 @@ public void Configure(TelemetryConfiguration configuration)
                 configuration.InstrumentationKey = this.applicationInsightsServiceOptions.InstrumentationKey;
             }
 
+            if (this.telemetryProcessorFactories.Count() > 0)
```

Comment:
Actually, perhaps this if statement is unnecessary. If the imported ```telemetryProcessorFactories``` is empty, then the foreach does nothing, but I think we should still call ```.Build()``` for the case where users have manually added processors to the ```TelemetryProcessorChainBuilder```. Someone needs to call ```.Build()``` and this might be the right place.